api/envoy/config/core/v3/base.proto

```diff
@@ -320,12 +320,35 @@ message HeaderValueOption {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.api.v2.core.HeaderValueOption";
 
+  // Describes the supported actions types to be used in :ref:`append_action <envoy_v3_api_field_config.core.v3.HeaderValueOption.append_action>`
+  enum HeaderAppendAction {
+    // This action will append the specified value to the existing values if the header
+    // already exist. If the header doesn't exist then this will add the header with
+    // specified key and value.
+    APPEND_IF_EXISTS = 0;
+
+    // This action will only append the header if it doesn't already exist. If the header
+    // already exist then this will be a no-op.
+    APPEND = 1;
```

Comment:
For clarity I would call this `ADD_IF_ABSENT` for clarity. That is what it is, right?